mongodb_atlas/metadata.csv

```diff
@@ -30,10 +30,10 @@ mongodb.atlas.system.cpu.mongoprocess.norm.childrenkernel,gauge,,percent,,Normal
 mongodb.atlas.system.cpu.mongoprocess.norm.childrenuser,gauge,,percent,,Normalized CPU children user space for mongodb processes,0,mongodb_atlas,normalized cpu children user space for mongodb processes,
 mongodb.atlas.system.cpu.mongoprocess.norm.kernel,gauge,,percent,,Normalized CPU kernel space for mongodb processes,0,mongodb_atlas,normalized cpu kernel space for mongodb processes,
 mongodb.atlas.system.cpu.mongoprocess.norm.user,gauge,,percent,,Normalized CPU user space for mongodb processes,0,mongodb_atlas,normalized cpu user space for mongodb processes,
-mongodb.atlas.system.cpu.mongoprocess.norm.max.childrenkernel,gauge,,percent,,Max normalized CPU children kernel space for mongodb processes,0,mongodb_atlas,max normalized cpu children kernel space for mongodb processes,
```

Comment:
What's the reason to modify this file?

Reply:
None. It was updated and I hadn't pulled for a while.

[lli-datadog] authored and [j8-redis] committed last month

https://github.com/redis-field-engineering/datadog-integrations-extras/commit/d94688fb6951c7d9ac244658dc2f04576874643c

How should I revert/undo this?

Reply:
you could use `git revert` or undo the changes manually, up to you.

Reply:
reverted